### PR TITLE
Update overview-and-concepts.mdx

### PR DESCRIPTION
**Pull request by ftouserkani-edb**
## What Changed?

Please review grammar as I have suggested a few changes/updates.

### CODE REVIEW

File: advocacy_docs/edb-postgres-ai/overview/overview-and-concepts.mdx [hunk 1]
```diff
@@ -11,25 +11,25 @@ EDB Postgres AI is composed of multiple elements which come together to deliver
...
 * **[Cloud Service Managed](/edb-postgres-ai/cloud-service/managed)**
-  * Bring your own cloud account to EDB and let the Managed environment take care of deploying and managing your Postgres databases in your cloud account
+  * Bring your own cloud account to EDB and let the Managed Service environment take care of deploying and managing your Postgres databases in your own cloud provider account
 
 ## [EDB Postgres AI Databases](/edb-postgres-ai/databases)
-All of EDB's database expertise can be found in EDB Postgres Advanced Server and EDB Postgres Extended Server. Oracle compatibility, transparent data encryption and more. They provide the data fabric on which EDB Postgres AI operates. Combined with EDB Postgres Distributed, they can also provide a high availability environment for your data. 
+All of EDB's database expertise can be found in EDB Postgres Advanced Server and EDB Postgres Extended Server. Oracle compatibility, TDE (Transparent Data Encryption) and more. They provide the data fabric on which EDB Postgres AI operates. Combined with EDB Postgres Distributed, they can also provide a high availability environment for your data.
```
Comment (ebgitelman) [inline advocacy_docs/edb-postgres-ai/overview/overview-and-concepts.mdx:28]:
```suggestion
All of EDB's database expertise can be found in EDB Postgres Advanced Server and EDB Postgres Extended Server, including Oracle compatibility, transparent data encryption (TDE), and more. They provide the data fabric on which EDB Postgres AI operates. Combined with EDB Postgres Distributed, they can also provide a high-availability environment for your data. 
```

File: advocacy_docs/edb-postgres-ai/overview/overview-and-concepts.mdx [hunk 2]
```diff
@@ -39,10 +39,10 @@ All of these components are available on the EDB Postgres AI Cloud Service, and
 Filtering out the data noise and revealing insights and value, Lakehouse analytics brings both structured relational data in Postgres and unstructured data in object storage together for exploration.
 
 * **[Lakehouse nodes](/edb-postgres-ai/analytics/lakehouse)**
-  * At the heart of Analytics is a custom built store for this data. Built to bring structured and unstructured data together, Lakehouse nodes support numerous formats to bring your data in from the cold, ready for analysis. 
+  * At the heart of Analytics is a custom built object storage for your data. Built to bring structured and unstructured data together, Lakehouse nodes support numerous formats to bring cold data in ready for analysis.
```
Comment (ebgitelman) [inline advocacy_docs/edb-postgres-ai/overview/overview-and-concepts.mdx:42]:
```suggestion
  * At the heart of Analytics is custom-built object storage for your data. Built to bring structured and unstructured data together, Lakehouse nodes support numerous formats to bring cold data in, ready for analysis. 
```

File: advocacy_docs/edb-postgres-ai/overview/overview-and-concepts.mdx [hunk 3]
```diff
@@ -39,10 +39,10 @@ All of these components are available on the EDB Postgres AI Cloud Service, and
...
 
 * **[Lakehouse nodes](/edb-postgres-ai/analytics/lakehouse)**
-  * At the heart of Analytics is a custom built store for this data. Built to bring structured and unstructured data together, Lakehouse nodes support numerous formats to bring your data in from the cold, ready for analysis. 
+  * At the heart of Analytics is a custom built object storage for your data. Built to bring structured and unstructured data together, Lakehouse nodes support numerous formats to bring cold data in ready for analysis. 
 
 ## [EDB Postgres AI AI/ML](/edb-postgres-ai/ai-ml)
-* Postgres has proven its capability as a flexible data environment, and Vector data, the core of generative AI, is already infused into EDB Postgres AI providing a platform for a range of practical and effective AI/ML solutions. A technical preview of this capability is available for the pgai extension.
+* Postgres has proven its capability as a flexible data environment, and with Vector data as the core of generative AI, is already infused into EDB Postgres AI providing a platform for a range of practical and effective AI/ML solutions. A technical preview of this capability is available for the Postgres "pgai" extension.
```
Comment (ebgitelman) [inline advocacy_docs/edb-postgres-ai/overview/overview-and-concepts.mdx:45]:
```suggestion
* Postgres has proven its capability as a flexible data environment. With vector data as the core of generative AI, ir's already infused into EDB Postgres AI, providing a platform for a range of practical and effective AI/ML solutions. A technical preview of this capability is available for the Postgres pgai extension.
```

File: advocacy_docs/edb-postgres-ai/overview/overview-and-concepts.mdx [hunk 4]
```diff
@@ -11,25 +11,25 @@ EDB Postgres AI is composed of multiple elements which come together to deliver
...
-All of EDB's database expertise can be found in EDB Postgres Advanced Server and EDB Postgres Extended Server. Oracle compatibility, transparent data encryption and more. They provide the data fabric on which EDB Postgres AI operates. Combined with EDB Postgres Distributed, they can also provide a high availability environment for your data. 
+All of EDB's database expertise can be found in EDB Postgres Advanced Server and EDB Postgres Extended Server. Oracle compatibility, TDE (Transparent Data Encryption) and more. They provide the data fabric on which EDB Postgres AI operates. Combined with EDB Postgres Distributed, they can also provide a high availability environment for your data. 
 
 All of these components are available on the EDB Postgres AI Cloud Service, and managed through the EDB Postgres AI Console.
 * **[EDB Postgres Advanced Server](/edb-postgres-ai/databases/epas)**
-  * EDB Postgres Advanced Server provides provide mission critical capabilities for your data, with EDB Postgres Advanced Server also providing Oracle compatibility.
+  * EDB Postgres Advanced Server provides mission critical capabilities for your data along with Oracle compatibility.
```
Comment (ebgitelman) [inline advocacy_docs/edb-postgres-ai/overview/overview-and-concepts.mdx:32]:
```suggestion
  * EDB Postgres Advanced Server provides mission-critical capabilities for your data along with Oracle compatibility.
```

File: advocacy_docs/edb-postgres-ai/overview/overview-and-concepts.mdx (hunk 3 above)
Comment (ebgitelman) [reply advocacy_docs/edb-postgres-ai/overview/overview-and-concepts.mdx:45]: Please check if this is correct. It was hard to follow.

**Review by ebgitelman — COMMENTED** [review]
I've reviewed the edits and had a couple of queries and comments.

File: advocacy_docs/edb-postgres-ai/overview/overview-and-concepts.mdx [hunk 5]
```diff
@@ -11,25 +11,25 @@ EDB Postgres AI is composed of multiple elements which come together to deliver
...
 * **[EDB Postgres AI Estate](/edb-postgres-ai/console/estate)**
-  * EDB Postgres AI Cloud Service databases are automatically managed on the Console. The Estate can also monitor the health of AWS RDS and S3 resources and using the Agent, on-premises databases show in the same view.
+  * EDB Postgres AI Cloud Service databases are automatically managed on the Console. The Estate can also monitor the health of AWS RDS and S3 resources and using the Agent, on-premise databases are shown and managed in the same view.
 * **[EDB Postgres AI Agent](/edb-postgres-ai/console/agent)**
-  * Include on premises databases under the Console's single pane of glass view with the Agent. The Agent is part of enabling EDB Postgres AI an unprecedented view of diverse deployments.
+  * Include on premise databases under the Console's single pane of glass view with the Agent. The Agent is part of enabling EDB Postgres AI an unprecedented view of diverse deployments.
```
[the remark this answers is not in the document]
Comment (djw-m) [reply advocacy_docs/edb-postgres-ai/overview/overview-and-concepts.mdx:18]: It would read better. 

```suggestion
  * Include on premise databases under the Console's single pane of glass view with the Agent. The Agent is part of giving EDB Postgres AI an unprecedented view of diverse deployments.
```

File: advocacy_docs/edb-postgres-ai/overview/overview-and-concepts.mdx (hunk 3 above)
Comment (djw-m) [reply advocacy_docs/edb-postgres-ai/overview/overview-and-concepts.mdx:45]:
```suggestion
* Postgres has proven its capability as a flexible data environment and EDB Postgres AI is already infused with Vector data support, enabling generative AI and providing a platform for a range of practical and effective AI/ML solutions. A technical preview of this capability is available for the Postgres "pgai" extension.
```

**Review by djw-m — APPROVED** [review]
LGTM
